setup.cfg

```diff
@@ -77,6 +77,7 @@ filterwarnings =
     ignore:coroutine 'Scheduler\.restart' was never awaited:RuntimeWarning
     ignore:coroutine 'Semaphore._refresh_leases' was never awaited:RuntimeWarning
     ignore:coroutine 'ServerSession\.with_document_locked' was never awaited
+    ignore:coroutine '.*\.write' was never awaited:RuntimeWarning
```

Comment:
I'd be tempted to not have this be quite such a widely applicable regex

Reply:
Yeah we pretty much only see this with `InProc`, but we've also seen it with TCP once or twice. I could just do those two?